### ISSUE TITLE
Atom: data set identifier namespace

### ISSUE THREAD

**Issue by heidivanparys** [inspire-eu-validation/download-service#89]
I have a question regarding test "[Download Service feed dataset identifiers](https://github.com/inspire-eu-validation/download-service/blob/3.1/atom-pre-defined/download-service-feed-dataset-identifiers.md)" .

As far as I can see from the TG Download Service (Table 13) and the TG Metadata v2.0 (page 35), the spatial_dataset_identifier_namespace is optional. Thus shouldn't Requirement 13 in the TG Download Service be updated to reflect this?

![billede](https://user-images.githubusercontent.com/11427611/40421894-f49d982e-5e7c-11e8-93e1-d50be4b947e9.png)

![billede](https://user-images.githubusercontent.com/11427611/40421981-463872ee-5e7d-11e8-8688-93023a336cdb.png)

![billede](https://user-images.githubusercontent.com/11427611/40422118-c902f064-5e7d-11e8-9992-c4500dce0a77.png)

**Comment by aquaglia** [comment]
Dear Heidi,
yes, I agree,


Best regards,
Angelo

**Comment by bgsmase** [comment]
Yes, agree.

**Comment by thijsbrentjens** [comment]
Agreed. Just to be sure: for WFS the namespace is not mandatory either, correct?

**Comment by heidivanparys** [comment]
Correct. The XML schema for the Extended Capabilities defines the namespace as optional:

![billede](https://user-images.githubusercontent.com/11427611/40485049-fed95bc2-5f4c-11e8-8d4e-ab691cb4562a.png)

See http://inspire.ec.europa.eu/schemas/inspire_dls/1.0/inspire_dls.xsd and http://inspire.ec.europa.eu/schemas/common/1.0/common.xsd

**Comment by thijsbrentjens** [comment]
This also means that requirement 50 and 51, for WFS Stored Queries should be updated. These are similar to TG Requirement 13.

**Comment by heidivanparys** [comment]
Change proposal created in https://github.com/INSPIRE-MIF/technical-guidelines/issues/82.
